nixos/modules/services/network-filesystems/ceph.nix

```diff
@@ -157,6 +163,27 @@ in
           A comma-separated list of subnets that will be used as cluster networks in the cluster.
         '';
       };
+
+      rgwMimeTypesFile = mkOption {
+        type = with types; nullOr path;
+        default = "${pkgs.mime-types}/etc/mime.types";
```

Comment:
This should use defaultText: https://github.com/NixOS/nixpkgs/blob/2e869f71a7aebf0e96114cf7317b73583e893eac/nixos/modules/config/krb5/default.nix#L86-L87